applications/main/infrared/infrared_cli.c

```diff
@@ -188,14 +195,11 @@ static bool infrared_cli_parse_raw(const char* str, InfraredSignal* signal) {
             ++str;
         }
 
-        char timing_str[INFRARED_CLI_BUF_SIZE];
-        if(sscanf(str, "%9s", timing_str) != 1) {
+        uint32_t timing;
+        if(strint_to_uint32(str, (char**)&str, &timing, 10) != StrintParseNoError) {
```

Comment:
Better use a local discardable variable than cast const away

Reply:
Seems fine in this context to be fair, it's just a limitation of C because you can't express "pointer to a pointer to char **or** const char as we don't care about the constness". `strtol` from the standard seems to face the same issue.

Reply:
@CookiePLMonster here is rabbit hole for you: https://c-faq.com/ansi/constmismatch.html